flink-table-store-connector/src/main/java/org/apache/flink/table/store/connector/source/FileStoreSource.java

```diff
@@ -79,6 +82,7 @@ public FileStoreSource(
             boolean isContinuous,
             long discoveryInterval,
             boolean latestContinuous,
+            boolean nonRescaleCompact,
```

Comment:
Can we keep this class as it is?
We can just create empty `FromElementsSource` for this?

Reply:
> FromElementsSource

`FromElementsSource` resides in package `flink-connector-test-utils`, is it proper to use?

Reply:
> > FromElementsSource
> 
> `FromElementsSource` resides in package `flink-connector-test-utils`, is it proper to use?

Or just impl a new source like `FromElementsSource`